static/app/views/replays/detail/console/useConsoleFilters.spec.tsx

```diff
@@ -78,29 +112,81 @@ const breadcrumbs = [
   },
 ];
 
-describe('replays/details/console', () => {
-  it('filters console breadcrumbs by log level', () => {
-    const result = filterBreadcrumbs(breadcrumbs, '', ['error', 'warning']);
-    expect(result.length).toEqual(2);
+describe('useConsoleFilters', () => {
+  beforeEach(() => {
+    mockBrowserHistoryPush.mockReset();
   });
 
-  it('filters console breadcrumbs by search term', () => {
-    const result = filterBreadcrumbs(breadcrumbs, 'comp', []);
-    expect(result.length).toEqual(2);
+  it('should update the url on user input', () => {
+    mockUseLocation.mockReturnValue({
+      pathname: '/',
+      query: {},
+    } as Location<Query>);
+
+    const {result} = reactHooks.renderHook(() => useConsoleFilters({breadcrumbs}));
+
+    result.current.setLogLevel(['error']);
+    expect(browserHistory.push).toHaveBeenLastCalledWith({
+      pathname: '/',
+      query: {
+        consoleLogLevel: ['error'],
+      },
+    });
+
+    result.current.setSearchTerm('component');
```

Comment:
what is `current`?

Reply:
`current` comes out of RTL, it's a ref (?) i guess that holds the last output of the hook: https://testing-library.com/docs/react-testing-library/api/#renderhook